scripts/check-etherscan-api.ts

```diff
@@ -0,0 +1,92 @@
+import { go } from '@api3/promise-utils';
+import { z } from 'zod';
+
+import { type Chain, CHAINS } from '../src/index';
+
+const ResultSchema = z.object({
+  chainname: z.string(),
+  chainid: z.string(),
+  blockexplorer: z.string(),
+  apiurl: z.string(),
+  status: z.number(),
+  comment: z.string(),
+});
+
+const EtherscanV2SupportListSchema = z.object({
```

Comment:
We defined schema here but utilized it only for inferring type `EtherscanV2SupportList`. I'd suggest using this to parse response from URL.